app/screens/login/login.js

```diff
@@ -299,15 +299,28 @@ class Login extends PureComponent {
                 />
             );
         } else {
+            const additionalStyle = {};
+            if (this.props.config.EmailLoginButtonColor) {
+                additionalStyle.backgroundColor = this.props.config.EmailLoginButtonColor;
```

Comment:
Shouldn't these colours come from the LocalConfig, not the server config?

Reply:
They originally were in the local config, but we decided to move them to the server side so that the webapp could take advantage of them as well.